apps/web/client/src/pages/_error.tsx

```
@@ -1,16 +1,82 @@
-function Error({ statusCode }: { statusCode: number }) {
+import type { NextPageContext } from 'next';
+import Link from 'next/link';
+
+interface ErrorProps {
+    statusCode: number;
+    hasGetInitialPropsRun?: boolean;
+    err?: Error;
+}
+
+function Error({ statusCode, hasGetInitialPropsRun, err }: ErrorProps) {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)